.github/workflows/release.yml

```diff
@@ -24,20 +25,29 @@ jobs:
         run: |
           python -m pip install --upgrade pipenv
           pipenv install --dev
-          mkdir "${GITHUB_WORKSPACE}/bin"
-          curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b "${GITHUB_WORKSPACE}/bin"
-          chmod +x "${GITHUB_WORKSPACE}/bin/syft"
-          echo "${GITHUB_WORKSPACE}/bin" >> "${GITHUB_PATH}"
-      - name: Install cosign
-        uses: sigstore/cosign-installer@main
+          mkdir "${RUNNER_TEMP}/bin"
+          curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b "${RUNNER_TEMP}/bin"
+          chmod +x "${RUNNER_TEMP}/bin/syft"
+          curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b "${RUNNER_TEMP}/bin"
+          chmod +x "${RUNNER_TEMP}/bin/grype"
+          echo "${RUNNER_TEMP}/bin" >> "${GITHUB_PATH}"
       - name: Build the images
-        run: pipenv run invoke build --debug
+        run: pipenv run invoke build
       - name: Generate SBOMs
-        run: pipenv run invoke sbom --debug
-      - uses: actions/upload-artifact@v2
+        run: pipenv run invoke sbom
+      - name: Upload SBOMs
+        uses: actions/upload-artifact@v2
         with:
           name: SBOMs
-          path: sbom.*.spdx.json
+          path: sbom.*.v*.json
```

Comment:
This gives us the release-only SBOM artifacts as a part of releases..  Same with the vuln scans below

Reply:
Necessary step!